include/ebpf_program_types.h

```diff
@@ -48,6 +48,7 @@ typedef struct _ebpf_helper_function_addresses
 {
     uint32_t helper_function_count;
     MIDL([size_is(helper_function_count)]) uint64_t* helper_function_address;
+    MIDL([size_is(helper_function_count)]) uint64_t* helper_function_flags;
```

Comment:
I still don't follow why this change is needed, since the service already gets the helper function return type and you can get the flag from that.